gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -172,7 +167,7 @@ def run_initdb
     end
 
     def start_postgres
-      LinuxAdmin::Service.new(PostgresAdmin.service_name).start
+      LinuxAdmin::Service.new(PostgresAdmin.service_name).enable.start
```

Comment:
I'll add a fork/detach to fix https://github.com/ManageIQ/manageiq/pull/10241/files#r73583958